File: app/src/main/java/andiag/coru/es/welegends/rest/entities/persist/ProfileIcon.java
package andiag.coru.es.welegends.rest.entities.persist;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by andyqm on 06/06/2016.
 */
public class ProfileIcon {

    private Integer id;
    private Image image;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Image getImage() {
        return image;
    }

    public void setImage(Image image) {
        this.image = image;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }
}

File: WeLegends/app/src/main/java/es/coru/andiag/welegends/models/entities/ProfileIcon.kt
package es.coru.andiag.welegends.models.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.orm.SugarRecord
import java.io.Serializable

/**
 * Created by Canalejas on 09/12/2016.
 */

class ProfileIcon : SugarRecord(), Serializable {

    @SerializedName("id") @Expose var key: Number? = null
    @Expose var image: Image? = null

}
